packages/main/src/components/FilterGroupItem/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { usePassThroughHtmlProps } from '@ui5/webcomponents-react-base/lib/usePassThroughHtmlProps';
+import { createComponentStyles } from '@ui5/webcomponents-react-base/lib/createComponentStyles';
+import { BusyIndicator } from '@ui5/webcomponents-react/lib/BusyIndicator';
+import { BusyIndicatorSize } from '@ui5/webcomponents-react/lib/BusyIndicatorSize';
+import { FlexBox } from '@ui5/webcomponents-react/lib/FlexBox';
+import { Label } from '@ui5/webcomponents-react/lib/Label';
+import React, { forwardRef, ReactElement, RefObject } from 'react';
+import { CommonProps } from '../../interfaces/CommonProps';
+import styles from './FilterGroupItem.jss';
+
+const useStyles = createComponentStyles(styles, { name: 'FilterGroupItem' });
+
+export interface FilterGroupItemPropTypes extends CommonProps {
+  children: ReactElement;
+  label?: string;
+  groupName?: string;
+  labelTooltip?: string;
+  loading?: boolean;
+  mandatory?: boolean;
+  visible?: boolean;
+  visibleInFilterBar?: boolean;
+  considerGroupName?: boolean;
+}
+
+export const FilterGroupItem = forwardRef((props: FilterGroupItemPropTypes, ref: RefObject<HTMLDivElement>) => {
+  const classes = useStyles();
+  const {
+    groupName,
+    considerGroupName,
+    label,
+    labelTooltip,
+    mandatory,
+    visible,
+    visibleInFilterBar,
+    children,
+    style,
+    loading,
+    // @ts-ignore
+    inFB
+  } = props;
+
+  const passThroughProps = usePassThroughHtmlProps(props);
+
+  if (!mandatory && (!visible || (inFB && !visibleInFilterBar))) return null;
+  return (
+    <div
+      ref={ref}
+      {...passThroughProps}
+      className={inFB ? classes.filterItem : classes.filterItemDialog}
+      style={inFB ? style : {}}
```

Comment:
you can create the empty object outside of this component and use the `stableEmptyObject` instead